sda/cmd/ingest/ingest.go

```diff
@@ -100,6 +101,24 @@ func main() {
 	var message schema.IngestionTrigger
 
 	go func() {
+		start := time.Now()
+		for i := 0; i <= 10; i++ {
+			h, err := db.ListKeyHashes()
+			if err != nil {
+				log.Errorln(err.Error())
+			}
+			if len(h) != 0 {
+				break
+			}
```

Comment:
```suggestion
			if err != nil {
				log.Errorln(err.Error())
			} else if len(h) != 0 {
				break
			}
```

Without if else, it assumes that ListKeyHashes() returns a valid slice even if an error occurs. This may lead to problem  in edge cases.

Reply:
`ListKeyHashes()` always returns a valid slice according to the DB tests.